docs/user_guides/fs/feature_group/on_demand_transformations.md

```diff
@@ -0,0 +1,180 @@
+
+### Accessing on-demand features in feature views
+
+A feature view can include on-demand features from feature groups by selecting them in the [query](../feature_view/query.md) used to create the feature view. These on-demand features can be utilized similarly to regular features, and [model-dependent transformations](../feature_view/model-dependent-transformations.md) can be applied to them if required.
+
+=== "Python"    
+!!! example "Creating feature view with on-demand features"
+    ```python
+
+    # Selecting on-demand features in query
+    query = fg.select(["id", "feature1", "feature2", "on_demand_feature3", "on_demand_feature4"])
+
+    # Creating a feature view using a query that containing on-demand transformations and model-dependent transformations
+    feature_view = fs.create_feature_view(
+            name='transactions_view',
+            query=query,
+            transformation_functions=[
+                min_max_scaler("feature1"),
+                min_max_scaler("on_demand_feature3"),
+            ]
+        )
+    ```
+
+### Recomputing on-demand features
+
+On-demand features in the feature view can be recomputed in real-time during online inference using the same on-demand transformation functions used to create them. Hopsworks, by default, automatically recomputes all on-demand features when retrieving feature vectors with the functions `get_feature_vector` and `get_feature_vectors`. Additionally, on-demand features can be recomputed using the `compute_on_demand_features` function or by manually executing the on-demand transformation functions.
+
+The values for the input parameters required to recompute on-demand features can be provided using the `request_parameters` argument. If values are not provided through the `request_parameters` argument, the transformation function will verify if the feature vector contains the necessary input parameters and will use those values instead. However, if the required input parameters are also not present in the feature vector, an error will be thrown.
+
+!!! note
+    By default the functions `get_feature_vector` and `get_feature_vectors` will apply model-dependent transformation present in the feature view after recomputing on-demand features.
+
+#### Retrieving feature vector
+
+The `get_feature_vector` function retrieves a single feature vector based on a serving key. The on-demand features in the feature vector can be recomputed using real-time data by passing a dictionary that associates the name of each input parameter needed for the on-demand transformation function with its respective new value to the `request_parameter` argument.
+
+=== "Python"    
+!!! example "Recomputing on-demand features while retrieving a feature vector"
+    ```python
+
+    feature_vector = feature_view.get_feature_vector(entry={"id":1}, request_parameter={"arg1":1, "arg2":2})
```

Comment:
maybe refer the argument's name to previous example i.e transaction_date, current_date?